.github/workflows/library.yml

```diff
@@ -104,24 +102,20 @@ jobs:
             task: "cypress"
           - os: "macos"
             task: "slow"
-          - os: "macos"
-            task: "binary"
           - os: "macos"
             task: "pyright"
           - os: "macos"
             task: "propagate"
 
         include:
           # Version numbers chosen so that highest and lowest Python version is
-          # utilised, 3.7 is chosen on Windows to align with Jupyter Book until
-          # https://github.com/executablebooks/jupyter-book/issues/906 is fixed.
-          # See https://jupyterbook.org/advanced/advanced.html#working-on-windows
+          # utilised
```

Comment:
would be useful to have explanation that says:
3.x is the latest version, so we use that unless...
Use 3.9 for Ubuntu because ...
Use 3.8 for MacOS because ...

Reply:
Yup, will do :slightly_smiling_face:

Reply:
Fixed in https://github.com/pymedphys/pymedphys/pull/1694/commits/9d87f719fafbc0af7f2a4d70f96d986fc89da450